src/character-card.js

```diff
@@ -0,0 +1,248 @@
+import React, {
+    Fragment,
+    useState,
+    useEffect,
+    useRef
+} from 'react';
+import styled from 'styled-components';
+
+import copy from 'clipboard-copy';
+
+import ClipboardSolid from './icons/ClipboardSolid';
+import PenSolid from './icons/PenSolid';
+import TrashSolid from './icons/TrashSolid';
+
+import {
+    AttributeGroupLabel,
+    CharacterCardColumn,
+    CharacterCardContainer,
+    CharacterCardRow,
+    CharacterCardToolbar,
+    NameWrapper,
+    ToolbarButton
+} from './styles.js';
+
+const capitalize = string => string && string.charAt(0).toUpperCase() + string.slice(1);
+
+const indefiniteArticleFor = string => {
+    const VOWELS = ['a', 'e', 'i', 'o', 'u'];
+    return VOWELS.includes(string.charAt(0)) ? 'an' : 'a';
+}
+
+// this logic should moved to the character module
+const displayGender = (age, gender) => {
+    switch (gender) {
+        case 'cis male': return ['young', 'teenage'].includes(age) ? 'boy' : 'man';
+        case 'cis female': return ['young', 'teenage'].includes(age) ? 'girl' : 'woman';
+        case 'trans male': return ['young', 'teenage'].includes(age) ? 'trans boy' : 'trans man';
+        case 'trans female': return ['young', 'teenage'].includes(age) ? 'trans girl' : 'trans woman';
+        case 'genderfluid': return ['young', 'teenage'].includes(age) ? 'genderfluid child' : 'genderfluid person';
+        case 'agender': return ['young', 'teenage'].includes(age) ? 'agender child' : 'agender person';
+        default: return 'person';
+    }
+}
+
+const AttributeGroupWrapper = styled.div`
```

Comment:
Moved the styled components that I edited next to the components that use them. This makes maintenance easier, but I didn't go all the way with it.